prow/config.yaml

```diff
@@ -193,6 +193,16 @@ presets:
           secretKeyRef:
             name: slack-nightly-token
             key: token
+  - labels:
```

Comment:
why you introduce new label, instead of using: preset-stability-checker-slack-notifications